src/data/dynamic/admin/Teams.js

```diff
@@ -62,8 +62,10 @@ export const COLUMNS = [
     cell: ({ getValue }) => (
       <div>
         {getValue().map(({ name, link }, index) => (
-          <Link key={index} href={link} className="mx-2">
-            {name}
+          <Link key={index} href={link} className="mx-2 inline-flex">
+            {name === "github" && ICONS.github}{" "}
```

Comment:
name is always a valid key so u can just do `icons[name]` and it should be working